cmd/talosctl/pkg/talos/action/tracker.go

```diff
@@ -189,6 +191,38 @@ func (a *Tracker) Run() error {
 		Backoff:           backoff.Config{},
 		MinConnectTimeout: 20 * time.Second,
 	}))
+
+	err = multierror.Append(err, eg.Wait())
+
+	if !a.debug {
+		return err
+	}
+
+	var failedNodes []string
+
+	failedNodesToDmesgs.Range(func(key, value any) bool {
+		failedNodes = append(failedNodes, key.(string))
+
+		return true
+	})
+
+	if len(failedNodes) > 0 {
+		sort.Strings(failedNodes)
+
+		fmt.Printf("console logs for nodes %v:\n", failedNodes)
+
+		for _, node := range failedNodes {
+			dmesgReaderRaw, _ := failedNodesToDmesgs.Load(node)
+			dmesgReader := dmesgReaderRaw.(io.Reader) //nolint:errcheck
+
+			_, copyErr := io.Copy(os.Stdout, dmesgReader)
+			if copyErr != nil {
+				fmt.Printf("%v: failed to print debug logs: %v\n", node, copyErr)
```

Comment:
```suggestion
				fmt.Printf("%q: failed to print debug logs: %v\n", node, copyErr)
```

Minor nit about quotes.

Reply:
not sure about this one because I see that for nodes, in more places we use `fmt.Printf("%s: ...", node, ...)` instead of `%q`

Reply:
Ok - both options work for me

Reply:
I like `%q` when it comes to empty strings, as it's confusing with `%v`